### ISSUE TITLE
Publish latest version to github

### ISSUE THREAD

**Issue by superandrew213** [shahen94/react-native-video-processing#185]
@shahen94 can you please publish latest version on here? Npm latest version is 1.13.0, on github it is still 1.7.1

**Comment by shahen94** [comment]
Will do in 1 week.

Thanks

**Comment by keshamtravels** [comment]
Please @shahen94  , I request you to do this fast , i have an urgent need of this in my project

**Comment by keshamtravels** [comment]
can you please tell me how to install this github version in to my project, I am unable to do so

**Comment by shahen94** [comment]
@keshamtravels 

In your package.json add this line

```
"react-native-video-processing": "git+https://github.com/shahen94/react-native-video-processing.git"
```

**Comment by shahen94** [comment]
Nevermind, I published the latest version to npm under the `1.20.0` version

**Comment by keshamtravels** [comment]
Thank you @shahen94  , for such a great support from your side
